### ISSUE TITLE
Concurrency tests not building?

### ISSUE THREAD

**Issue by dgsmith** [facebook/folly#829]
Trying to debug a `ConcurrentHashMap` crash I'm seeing. Was going to look at folly's tests for the `ConcurrentHashMap` but it seems as though they are not building using `make && make check`? Is there another way to build + run tests? Running on macOS, using latest master (`e9a27ccd5e820000231b29f0cad21bc01154121a`)

Thanks!

**Comment by dgsmith** [comment]
Also don't see tests being run on[ latest Travis build](https://travis-ci.org/facebook/folly/jobs/370367427)

**Comment by dgsmith** [comment]
As an update, the crash cannot be replicated with the proper build setup. However, it's unclear how to get the Folly tests for the concurrent hash map to build + run 🤷‍♀️

**Comment by Orvid** [comment]
TLDR; We have 2 (Well, currently 3, because the autotools version isn't gone yet) build systems used to build Folly, and the one we have internally had the concurrency tests enabled, but neither of the open-source ones had them enabled :(

I've got a diff up internally to add the concurrency tests to the CMake build, which should hopefully land in the next day or two.

**Comment by dgsmith** [comment]
Great, thanks!

**Comment by Orvid** [comment]
https://github.com/facebook/folly/commit/24eaeed1f92363b5f968a7b651518b034b7dc41f added the concurrency tests to the CMake build.
